content/docs/concepts/traffic-management/overview/index.md

```diff
@@ -0,0 +1,155 @@
+---
+title: Overview and Terminology
+description: Learn about Pilot, Istio's core traffic management component, Envoy proxies, and how they enable service discovery and load balancing.
+weight: 1
+keywords: [traffic-management, pilot, envoy-proxies, service-discovery, load-balancing]
+---
+
+With Istio, you can manage [traffic routing](/docs/concepts/traffic-management/routing)
+and [load balancing](/docs/concepts/traffic-management/overview/#load-balancing)
+for your service mesh without having to update your services. Istio simplifies
```

Comment:
Services is a Kubernetes word with a meaning.  You do have to update definitions in Istio which are called Services, but I think you mean the code underneath - want to call that out?

Reply:
Istio docs should (and hopefully do) explicitly say `Kubernetes service` if that's what it's referring to.

Reply:
We have [issue 3579](https://github.com/istio/istio.io/issues/3573) filed to improve the glossary to be able to not only make these distinctions but also provide our readers with definitions. Read more about it here: https://discuss.istio.io/t/improving-the-istio-glossary/2412